workflows/DNA-mapping/DNA-mapping

```diff
@@ -317,9 +315,9 @@ def main():
     ## create local temp dir and add this path to environment as $TMPDIR variable
     ## on SLURM: $TMPDIR is set, created and removed by SlurmEasy on cluster node
     temp_path = cf.make_temp_dir(args.tempdir, args.outdir, args.verbose)
-    snakemake_exports = ("export TMPDIR="+temp_path+" && ").split()
+    snakemake_exports += ("export TMPDIR="+temp_path+" && ").split()
```

Comment:
FYI, this will break if the path contains a space.

Reply:
It's fine for now, but in the future we might use something from `shlex` instead.